tests/flow/test_persistency.py

```diff
@@ -203,3 +204,29 @@ def test04_repeated_edges(self):
             actual_result = graph.query(q)
             self.env.assertEquals(actual_result.result_set, expected_result)
 
+        # Test decoding of graph meta keys in different sequences.
```

Comment:
In Replica-Of it is possible that queries will arrive while the dataset is still being loaded, please make sure we're handling this case gracefully.

Reply:
What will happen if `BGSAVE` is invoked while data is loaded?